bakery/checkers/namespace.go

```diff
@@ -18,6 +18,26 @@ type Namespace struct {
 	uriToPrefix map[string]string
 }
 
+// Equal reports whether ns2 encodes the same namespace
+// as the receiver.
+func (ns1 *Namespace) Equal(ns2 *Namespace) bool {
+	if ns1 == ns2 {
+		return true
+	}
+	if ns1 == nil || ns2 == nil {
+		return (ns1 == nil) == (ns2 == nil)
```

Comment:
wouldn't `return false` suffice here? the True case will be caught above.

Reply:
good point. i folded it into the test above.